tools/kagami/src/main.rs

```diff
@@ -95,16 +96,25 @@ mod crypto {
         #[clap(long, short, group = "generate_from")]
         seed: Option<String>,
         /// Output the key-pair in JSON format
-        #[clap(long, short)]
+        #[clap(long, short, group = "format")]
         json: bool,
+        /// Output the key-pair without additional text
+        #[clap(long, short, group = "format")]
+        compact: bool,
```

Comment:
I'm all for this format, but IMO the fact that we need to do some `bash` scripting to use this information as part of JSON means that the format probably needs adjustment. Most people would assume that you can just assign the output of kagami to an environment variable and be done with it.